src/server.js

```diff
@@ -1,14 +1,26 @@
 //Create a basic express server
 const express = require("express");
 const app = express();
+const mongoose = require("mongoose");
+
 require("dotenv").config();
+
 const port = process.env.SERVER_PORT || 3000;
+const db = process.env.DB_URL || "mongodb://localhost:27017/tyta";
+
+//Implement mongoose
+mongoose
+	.connect(db)
+	.then(() => console.log("Connected to MongoDB successfully"))
```

Comment:
Wouldn't we put the load of the routes in the then to avoid breaking all the roads if we can't connect to the Mongo db? With a "Database error" as the default result in case of an error

Reply:
```mongoose
	.connect(db)
	.then(() => {
		console.log("Connected to MongoDB");
		//Create a basic route
		app.get("/", (req, res) => {
			res.send("Hello World!");
		});

		app.use(express.json());

		//Start the server
		app.listen(port, () => {
			console.log(`Server listening at http://localhost:${port}`);
		});
	})
	.catch((err) => console.log("Could not connect to MongoDB", err));
```
Something like that ?

Reply:
Yes? I don't know if it's really a good practice, what do you think?

Reply:
I don't usually block the server if the connection fail (for exemple if we implement the swagger it will be blocked too?) but let's wait the others reviews